sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -99,6 +101,8 @@ def update_table_mux_status_for_statedb_port_tbl(table_name, status, read_side,
 def y_cable_toggle_mux_torA(physical_port):
     update_status = y_cable.toggle_mux_to_torA(physical_port)
     if update_status is True:
+        helper_logger.log_debug("Y_CABLE_DEBUG: Successful in toggling mux to ToR A for port {}".format(physical_port))
```

Comment:
Do we also want to log on failures?

Reply:
in the current implementation failures should be logged with y_cable_base.py/y_cable.py. But I think it would be good idea to log them for debugging as well. Let me add them.

Reply:
fixed